recipe/meta.yaml

```diff
@@ -42,6 +45,7 @@ test:
     imports:
         - iris
         - iris.fileformats.pp_packing  # [not win]
+        - iris.fileformats._pyke_rules.compiled_krb  # [not win]
```

Comment:
@pelson On your PR you forgot to re-render. So I absorbed you changes here.
Also, this import was failing on Windows there. Not sure if that is supposed to work on Windows.